Userland/Utilities/file.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2021, Valtteri Koskivuori <vkoskiv@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/MappedFile.h>
+#include <AK/Vector.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/FileStream.h>
+#include <LibCore/MimeData.h>
+#include <LibGfx/ImageDecoder.h>
+#include <stdio.h>
+#include <unistd.h>
+
+static Optional<String> placeholder(String description, String& path)
+{
+    (void)path;
+    return description;
+}
```

Comment:
We usually use `[[maybe_unused]]` instead of `(void)`'ing unused arguments. Also I think something like `description_only` is a slightly better name for this.

```suggestion
static Optional<String> description_only(String description, [[maybe_unused]] const String& path)
{
    return description;
}
```